xbmc/addons/AddonCallbacksPVR.cpp

```diff
@@ -150,7 +150,8 @@ void CAddonCallbacksPVR::PVRTransferEpgEntry(void *addonData, const ADDON_HANDLE
   }
 
   /* transfer this entry to the epg */
-  xbmcEpg->UpdateEntry(epgentry, handle->dataIdentifier == 1 /* update db */);
+  CEpgInfoTag tag(*epgentry);
+  xbmcEpg->UpdateEntry(tag);
```

Comment:
I'd opt for using a CEpgInfoTagPtr here, for consistency reasons. We use shared pointers whereever possible since my optimization PRs got integrated.